robottelo/utils/issue_handlers/jira.py

```diff
@@ -272,3 +272,41 @@ def get_default_jira(number):  # pragma: no cover
         "resolution": "",
         "error": "missing jira api_key",
     }
+
+
+def add_comment_on_jira(
+    jira_number,
+    comment,
+    comment_type=settings.jira.comment_type,
+    comment_visibility=settings.jira.comment_visibility,
+):
+    """Adds a new comment to a Jira issue.
+
+    Arguments:
+        jira_numbers {str} -- Jira issue number, ex. SAT-12232
+        comment {str}  -- Comment to add on the issue.
+        comment_type {str}  -- Type of comment to add.
+        comment_visibility {str}  -- Comment visibility.
+
+    Returns:
+        [list of dicts] -- [{'id':..., 'status':..., 'resolution': ...}]
+    """
+    if not settings.jira.enable_comment:
+        logger.warning(
```

Comment:
I think and info level is more appropriate here, also there is no return so it will continue to try to comment.

Reply:
As you stated, we are missing `return` here.
I am in favor of keeping the warning level, but if we want to keep it, I would slightly modify the condition.
```python
if settings.jira.enable_comment ^ pytest_option_enable_comment:
```
But this requires `pytest_option_enable_comment` to be available in the context of this function. That could be achieved through settings or a function kwarg.

Reply:
updated the PR. @ogajduse I'm in favor of keeping the warning level too.